Client/Frontend/Browser/ButtonToast.swift

```diff
@@ -15,6 +15,18 @@ struct ButtonToastUX {
     static let ToastButtonBorderWidth: CGFloat = 1
 }
 
+private class CustomButton : UIButton {
```

Comment:
Nit: I would rename this to something more specific like `HighlightableButton`. `CustomButton` seems too generic.